modules/angular-meteor-collection.js

```diff
@@ -53,14 +54,61 @@ angularMeteorCollection.factory('AngularMeteorCollection', [
       return allPromise;
     };
 
-    AngularMeteorCollection.remove = function (keys) {
-      // remove collection
-      if (!keys) _.pluck(this, '_id');
-      // remove single doc
-      keys = [].concat(keys);
+    AngularMeteorCollection._upsertDoc = function(doc, useUnsetModifier) { 
```

Comment:
@DAB0mB But why _upsertDoc?  AngularMeteorCollection's ````save```` does same thing.
I think here https://github.com/barbatus/angular-meteor/blob/master/modules/angular-meteor-object.js#L21 should be ````$meteor.collection(collection)```` and then at the object's save method correct to ````$$collection.save(updates)````